core/src/test/java/io/snyk/plugins/artifactory/scanner/ScannerModuleTest.java

```diff
@@ -37,31 +39,32 @@ void testGetScannerForPackageType() {
       mock(Repositories.class),
       mock(SnykClient.class));
 
-    assertTrue(
-      sm.getScannerForPackageType("myArtifact.jar").get()
-        instanceof MavenScanner);
-
-    assertTrue(
-      sm.getScannerForPackageType("myArtifact.tgz").get()
-        instanceof NpmScanner);
-
-    assertTrue(
-      sm.getScannerForPackageType("myArtifact.whl").get()
-        instanceof PythonScanner);
-
-    assertTrue(
-      sm.getScannerForPackageType("myArtifact.tar.gz").get()
-        instanceof PythonScanner);
+    assertEquals(MavenScanner.class, sm.getScannerForPackageType("myArtifact.jar").getClass());
+    assertEquals(NpmScanner.class, sm.getScannerForPackageType("myArtifact.tgz").getClass());
+    assertEquals(PythonScanner.class, sm.getScannerForPackageType("myArtifact.whl").getClass());
+    assertEquals(PythonScanner.class, sm.getScannerForPackageType("myArtifact.tar.gz").getClass());
+    assertEquals(PythonScanner.class, sm.getScannerForPackageType("myArtifact.zip").getClass());
+    assertEquals(PythonScanner.class, sm.getScannerForPackageType("myArtifact.egg").getClass());
```

Comment:
Refactored these to use `assertEquals` so assertion failures can say what the actual class is instead of just "expected true but got false"